src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/M2FiltersArgumentCollection.java

```diff
@@ -137,7 +137,9 @@
     public static final String LONG_INDEL_LENGTH_LONG_NAME = "long-indel-length";
 
     private static final double DEFAULT_LOG_10_PRIOR_OF_SNV = -6.0;
+    private static final double DEFAULT_LOG_10_PRIOR_OF_SNV_FOR_MITO = -2.5;
     private static final double DEFAULT_LOG_10_PRIOR_OF_INDEL = -7.0;
+    private static final double DEFAULT_LOG_10_PRIOR_OF_INDEL_FOR_MITO = -3.75;
```

Comment:
These are close-ish to the autosomal priors -- where did they come from?

Reply:
Back of the envelope calculation. I'll add a comment.